recipes-domu/domu-image-android/files/meta-xt-images-extra/recipes-extended/android/android.bb

```diff
@@ -24,9 +24,30 @@ ANDROID_VARIANT ?= "eng"
 SOC_FAMILY ?= "default"
 
 ANDROID_OUT_DIR_COMMON_BASE = ""
-DDK_KM_PREBUILT_MODULE ?= ""
-TARGET_PREBUILT_KERNEL ?= ""
-DDK_UM_PREBUILDS ?= ""
+
+python __anonymous () {
+    def create_param_str():
+         import os
```

Comment:
AFAIK, bitbake imports `os` automatically:

https://www.yoctoproject.org/docs/current/bitbake-user-manual/bitbake-user-manual.html#python-functions:

"The "bb" and "os" Python modules are automatically available. You do not need to import them."